specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-11-02-preview/examples/ScheduledEventsGet.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parameters": {
+    "api-version": "2023-11-02-preview",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "scheduledEventName": "event1"
+  },
+  "responses": {
```

Comment:
I think no need for this (not very helpful as you have it) sample if you remove the GET operation.

Reply:
Removed.